src/workerd/api/tracing-module.c++

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) 2017-2025 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#include "tracing-module.h"
+
+#include "src/workerd/jsg/inspector.h"
+
+#include <workerd/api/actor-state.h>
```

Comment:
What are inspector and actor-state used for? Add a comment if they are actually needed.